### PR TITLE
google-http-client: add STATUS_CODE_ACCEPTED to the list of status codes

### PR DESCRIPTION
**Pull request by mgenov**
Added constant for STATUS_CODE_ACCEPTED in the HttpStatusCodes interface.

### CODE REVIEW

**Comment by jadekler** [comment]
Hello - thanks for contributing. Just a quick update that this repository is currently frozen whilst we fix some backend stuff related to the release process, and submit a release. It should be unfrozen in a few weeks.

More generally, this repository is in maintenance mode; we may not be able to get to this PR for some time.

**Comment by mgenov** [comment]
@jadekler thanks for your feedback. Will wait till the repository is unfrozen.

**Comment by chingor13** [comment]
@mgenov Thanks for contributing!

The linter is failing on the added comment line being too long (over 100 characters). If you can fix it, then we'll get this merged.

File: google-http-client/src/main/java/com/google/api/client/http/HttpStatusCodes.java [hunk 1]
```diff
@@ -29,6 +29,9 @@
   /** Status code for a successful request that has been fulfilled to create a new resource. */
   public static final int STATUS_CODE_CREATED = 201;
 
+  /** Status code for a successful request that the request has been received but not yet acted upon. */
```
Comment (chingor13) [inline google-http-client/src/main/java/com/google/api/client/http/HttpStatusCodes.java]: This line is failing lint as it's >100 characters.

Comment (mgenov) [reply google-http-client/src/main/java/com/google/api/client/http/HttpStatusCodes.java]: Changes applied and the CI seems passing now.

**Pull request merged**